configuration/src/main/kotlin/com/malinskiy/marathon/config/OutputConfiguration.kt

```diff
@@ -2,6 +2,10 @@ package com.malinskiy.marathon.config
 
 import com.fasterxml.jackson.annotation.JsonProperty
 
+const val OUTPUT_MAX_PATH = 0
```

Comment:
It would be better to add the comment: "Value 0 is equivalent to unlimited file path length"

Reply:
Fixed